docs/source/utils.rst

```diff
@@ -13,4 +13,5 @@ ogusa.utils
 .. automodule:: ogusa.utils
   :members: mkdirs, pct_diff_func, convex_combo, read_file, pickle_file_compare,
     comp_array, comp_scalar, dict_compare, to_timepath_shape, get_initial_path,
-    safe_read_pickle, save_return_table
+    safe_read_pickle, rate_conversion, save_return_table, Inequality,
```

Comment:
Ah, I had forgotten to add `Inequality` when it was created.  The way to add the inequality class would be to add

```
.. autoclass:: Inequality
  :members: gini, var_of_logs, ratio_pct1_pct2, top_share
```

This should be at the same level of indentation as `automodule`.